packages/rpc-transport/src/transports/websocket/websocket-transport.ts

```diff
@@ -8,6 +8,10 @@ type Config = Readonly<{
 export function createWebSocketTransport({ url }: Config): IRpcWebSocketTransport {
     return async function ({ signal }: Parameters<IRpcWebSocketTransport>[0]) {
         signal.throwIfAborted();
-        return (await createWebSocketConnection({ signal, url })) as RpcWebSocketConnection;
+        return (await createWebSocketConnection({
+            sendBufferHighWatermark: 0,
```

Comment:
I really don't know what the right number is for this. I can't find _anything_ that says how large Node or the browser's buffer is (ie. how much `send()` data it will buffer before killing the `WebSocket`).

Reply:
I don't see much either, except for some huge numbers:

https://itecnote.com/tecnote/node-js-way-to-limit-the-maximum-size-of-a-websocket-message/
https://stackoverflow.com/questions/24153996/is-there-a-limit-on-the-size-of-a-string-in-json-with-node-js/24154610#24154610